src/database.rs

```diff
@@ -513,6 +516,31 @@ impl Database {
         self.verify.as_ref().unwrap()
     }
 
+    /// Get a statement by label.
+    pub fn statement(&mut self, name: &str) -> Option<StatementRef> {
+        match self.name_result().lookup_label(name.as_bytes()) {
+            None => None,
+            Some(lookup) => Some(self.parse_result().statement(lookup.address)),
+        }
+    }
+
+    /// Export an mmp file for a given statement.
+    pub fn export(&mut self, stmt: String) {
+        time(&self.options.clone(), "export", || {
+            let parse = self.parse_result().clone();
```

Comment:
I'm not a big fan of interior mutability, `RefCell`s and all that, because they just offload memory safety checking to runtime, when we could be doing better things with our time. Rust is geared toward compile-time memory safety, and I prefer that when possible.